domain/src/main/java/com/globant/domain/entity/PokemonCardDetails.kt

```diff
@@ -5,8 +5,8 @@ class PokemonCardDetails(
     var evolvesFrom: String?,
     var healthPoints: String?,
     var number: String,
-    var artist: String,
-    var rarity: String,
+    var artist: String?,
```

Comment:
are these **String?** necessary?  if comes in null just put there an empty string to avoid the null value

Reply:
it's wasn't necessary, refactored to avoid null values, thanks